### PR TITLE
✨ Combobox

### LINKED ISSUE #1555: Introduce Combobox react component
This is a merger of `SingleSelect` & `MultiSelect` to better support future feature requests such as #1430 & #1416

### PR DESCRIPTION
**Pull request by mimarz**
resolves #1555

### CODE REVIEW

File: libraries/core-react/src/components/Combobox/Combobox.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+  <Combobox label="Choose an element" meta="km/t" items={items} disabled />
+)
+
+export const ReadOnly: Story<ComboboxProps> = () => (
+  <Combobox label="This is read only" items={items} readOnly></Combobox>
+)
+
+const initial = ['Troms og Finnmark', 'Vestland']
+
+export const WithPreselected: Story<ComboboxProps> = () => (
```
[the remark this answers is not in the document]
Comment (pomfrida) [reply libraries/core-react/src/components/Combobox/Combobox.stories.tsx:50]: Just an empty input without `2/11 selected` Screenshot 2021-09-15 at 14 58 12

File: libraries/core-react/src/components/Combobox/Combobox.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+  <Wrapper>
+    <Combobox
+      label="I have preselected options"
+      items={items}
+      initialSelectedItems={initial}
+    />
+  </Wrapper>
+)
+
+export const UseOnChangeHandler: Story<ComboboxProps> = () => {
```
Comment (pomfrida) [inline libraries/core-react/src/components/Combobox/Combobox.stories.tsx:60]: Storybook crashes when testing the combobox in this example

File: libraries/core-react/src/components/Combobox/Combobox.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+      <Combobox
+        label="I can show my values outside the component itself"
+        items={items}
+        initialSelectedItems={initial}
+        handleSelectedItemsChange={handleSelectedItemsChange}
+      />
+    </Wrapper>
+  )
+}
+export const Controlled: Story<ComboboxProps> = () => {
```
Comment (pomfrida) [inline libraries/core-react/src/components/Combobox/Combobox.stories.tsx:82]: Crashes here too

File: libraries/core-react/src/components/Combobox/Combobox.stories.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (mimarz) [reply libraries/core-react/src/components/Combobox/Combobox.stories.tsx:57]: Fixed!

File: libraries/core-react/src/components/Combobox/Combobox.stories.tsx (hunk 2 above)
Comment (mimarz) [reply libraries/core-react/src/components/Combobox/Combobox.stories.tsx:68]: Fixed!

File: libraries/core-react/src/components/Combobox/Combobox.stories.tsx (hunk 3 above)
Comment (mimarz) [reply libraries/core-react/src/components/Combobox/Combobox.stories.tsx:89]: Fixed!

**Review by pomfrida — APPROVED** [review]
LGTM 👍

**Pull request merged**